quickstart.rst

```diff
@@ -45,6 +45,20 @@ Launch ArangoDB
    may need to use an alternate command: ``ARANGO_DATA=~/.local/multinet/arango
    docker-compose up``.) If you're getting an error that looks like ``docker-compose: 
    command not found``, make sure that the docker application is running.
+4. When the container is running, you'll need to make a new user on ArangoDB. To do so,
+   navigate to http://localhost:8529 and log in with the default username and password,
+   "root" and "letmein", respectively.
+5. Now that you're logged in, navigate to the "_system" database and then to "users" on
+   the left panel in the dashboard.
+6. Click "Add User" setting username to "readonly" and password to "letmein". Click
+   create to make the user.
+7. Now click on the user readonly to set its permissions. After clicking readonly, click
+   "permissions". On the permissions page, on the line that starts with "*", click the radio
```

Comment:
Github seems to think the `*` is the beginning of an italicized phrase (the text following it here in the diff view is italic) but the renderer treats it properly. I wonder if you want to go ahead and escape the `*` here--but this is totally non-urgent.

Reply:
Let's see how it renders on readthedocs and if it also sees it as italicized, I'll send another PR to escape it